playground/nitro.config.ts

```diff
@@ -1 +1,14 @@
-export default defineNitroConfig({});
+import { defineEventHandler } from "h3";
+
+export default defineNitroConfig({
+  modules: [
+    (_, nitro) => {
```

Comment:
Having resolved options first for Nuxt was mostly a backyard compatibility. For nitro we can simply accept `(nitro)`